code/modules/mining/ores_coins.dm

```diff
@@ -369,6 +369,7 @@
 	return w_type
 
 /obj/item/weapon/coin/is_screwdriver(var/mob/user)
+	to_chat(user, "<span class='warning'>You press the [src] against the screws forcefully!</span>")
```

Comment:
This will show the message no matter what their intent is, so it will even send this message if you are not using the coin as a screwdriver.

Reply:
I've added and tested an if check, I hope it is an acceptable solution.